tests/Fixer/Basic/NonPrintableCharacterFixerTest.php

```diff
@@ -232,131 +232,131 @@ function f(string $p)
         ];
 
         yield [
-            <<<'EXPECTED'
```

Comment:
This is a case where specific markers vastly saved me time identifying input vs. output of test cases. I would not change those.

Reply:
I already see a Fixer that is doing that for all data providers :D